### DISCUSSION TITLE
开启隧道时拒绝连接

### DISCUSSION THREAD

**Discussion by nbs07** [natfrp/wiki#179] [使用问题]
![1665146995702](https://user-images.githubusercontent.com/61622184/194557690-ac7cf67b-141e-40b4-9b98-dff9e7ed1bd3.png)
我用笔记本可以正常启动，这个台式不行，不知道什么原因

**Comment by tidymoon** [comment]
请检查 文件/frpc 权限问题

**Reply by KiKumo2776** [reply]
该怎么做？
